### PR TITLE
[DO NOT MERGE] Revert removal of Javascript tracking images

### PR DESCRIPTION
**Pull request by mobaig**
Revert "Revert "Merge pull request #452 from alphagov/add-non-js-tracking""
This reverts commit 2780a5367364167b154644ce441e27c6b11fb8a4.

We're re-running this experiment, so let's add back these images and
their tags.

https://trello.com/c/ee87DdP0/350-determine-how-many-people-have-don-t-have-javascript

Dependent on https://github.com/alphagov/govuk-puppet/pull/5441

### CODE REVIEW

File: app/views/homepage/index.html.erb [hunk 1]
```diff
@@ -229,3 +229,11 @@
     </div>
   </div>
 </main>
+<% # This is for JS/non-JS tracking purposes %>
```
Comment (nickcolley) [inline app/views/homepage/index.html.erb]: Should we make this a HTML comment so view-source adventurers know why we're doing shading looking tracking?

Comment (mobaig) [inline app/views/homepage/index.html.erb]: Updated.

**Comment by carolinegreen** [comment]
👍 Looks good to go, when ready.

**Comment by fofr** [comment]
When this was removed the experiment wasn't working end to end. It had never been configured with Fastly to work correctly.

The tracking code was on the site for a long time but not gathering data.

**Comment by carolinegreen** [comment]
@fofr Hi, we're trying to run the experiment again, and have it configured with Fastly.  How did it work originally if it was run but was never configured with Fastly in the first place?

**Comment by fofr** [comment]
We ran it before switching to Fastly. Was configured on old CDN.

**Comment by carolinegreen** [comment]
So it should be okay then, as we're going to configure it Fastly as well?

**Comment by mobaig** [comment]
Closing for now: https://trello.com/c/ee87DdP0/350-determine-how-many-people-have-don-t-have-javascript#comment-58a2dfb0b53567cacc648184

**Pull request closed without merging**